### ISSUE TITLE
graphQLErrors: Cannot return null for non-nullable field GameRelease.platform

### ISSUE THREAD

**Issue by bfalala** [andrew-codes/playnite-web#649]
Hi,

I started from scratch again with the latest version and cannot get my list of games to show up on the page.
The following error log is returned.

```
2024/11/30 18:15:58	stderr	} undefined
2024/11/30 18:15:58	stderr	  }
2024/11/30 18:15:58	stderr	    extensions: [Object: null prototype] {}
2024/11/30 18:15:58	stderr	    locations: undefined,
2024/11/30 18:15:58	stderr	    path: [ 'games', 588, 'releases', 1, 'platform' ],
2024/11/30 18:15:58	stderr	      at async Promise.all (index 0) {
2024/11/30 18:15:58	stderr	      at async Promise.all (index 588)
2024/11/30 18:15:58	stderr	      at async Promise.all (index 4)
2024/11/30 18:15:58	stderr	      at async Promise.all (index 1)
2024/11/30 18:15:58	stderr	      at async Promise.all (index 1)
2024/11/30 18:15:58	stderr	      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2024/11/30 18:15:58	stderr	      at /opt/playnite-web-app/server.production.js:188:5251
2024/11/30 18:15:58	stderr	      at tD (/opt/playnite-web-app/server.production.js:188:5905)
2024/11/30 18:15:58	stderr	  cause: Error: Cannot return null for non-nullable field GameRelease.platform.
2024/11/30 18:15:58	stderr	  extraInfo: undefined,
2024/11/30 18:15:58	stderr	  networkError: null,
2024/11/30 18:15:58	stderr	  clientErrors: [],
2024/11/30 18:15:58	stderr	  protocolErrors: [],
2024/11/30 18:15:58	stderr	  ],
2024/11/30 18:15:58	stderr	    }
2024/11/30 18:15:58	stderr	      extensions: [Object: null prototype] {}
2024/11/30 18:15:58	stderr	      locations: undefined,
2024/11/30 18:15:58	stderr	      path: [Array],
2024/11/30 18:15:58	stderr	        at async Promise.all (index 0) {
2024/11/30 18:15:58	stderr	        at async Promise.all (index 588)
2024/11/30 18:15:58	stderr	        at async Promise.all (index 4)
2024/11/30 18:15:58	stderr	        at async Promise.all (index 1)
2024/11/30 18:15:58	stderr	        at async Promise.all (index 1)
2024/11/30 18:15:58	stderr	        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
2024/11/30 18:15:58	stderr	        at /opt/playnite-web-app/server.production.js:188:5251
2024/11/30 18:15:58	stderr	        at tD (/opt/playnite-web-app/server.production.js:188:5905)
2024/11/30 18:15:58	stderr	    Error: Cannot return null for non-nullable field GameRelease.platform.
2024/11/30 18:15:58	stderr	  graphQLErrors: [
2024/11/30 18:15:58	stderr	    at e.value (/opt/playnite-web-app/server.production.js:452:201808) {
2024/11/30 18:15:58	stderr	    at yku (/opt/playnite-web-app/server.production.js:452:201266)
2024/11/30 18:15:58	stderr	    at Qr1 (/opt/playnite-web-app/server.production.js:452:200850)
2024/11/30 18:15:58	stderr	    at Object.next (/opt/playnite-web-app/server.production.js:454:21215)
2024/11/30 18:15:58	stderr	    at Object.then (/opt/playnite-web-app/server.production.js:454:21087)
2024/11/30 18:15:58	stderr	    at new Promise (<anonymous>)
2024/11/30 18:15:58	stderr	    at /opt/playnite-web-app/server.production.js:454:21120
2024/11/30 18:15:58	stderr	    at m (/opt/playnite-web-app/server.production.js:454:21205)
2024/11/30 18:15:58	stderr	    at /opt/playnite-web-app/server.production.js:459:73617
2024/11/30 18:15:58	stderr	    at new r (/opt/playnite-web-app/server.production.js:455:162)
2024/11/30 18:15:56	stderr	r [ApolloError]: Cannot return null for non-nullable field GameRelease.platform.
```

**Comment by andrew-codes** [comment]
I'm investigating this.

**Comment by andrew-codes** [comment]
@bfalala . Try dropping your database and using version ^10.0.0. I suspect some of the sync messages may have been dropping and causing this issue.

**Comment by bfalala** [comment]
Yes the issue is resolved in v10.0.0 but this issue came back:
[https://github.com/andrew-codes/playnite-web/issues/608](platform image undefined)
Any idea why ?

**Comment by andrew-codes** [comment]
I do have a suspicion as to why. I'll close this issue and we can track the missing platform icons on the other issue; of which I'll re-open.
